src/domain/individualCustomer/boatsCard/boatsCardUtils.ts

```diff
@@ -0,0 +1,32 @@
+import { BoatCertificateType } from '../../../@types/__generated__/globalTypes';
+import { Boat, BoatCert, LargeBoat } from '../types';
+
+const LARGE_BOAT_ID = '8';
+
+export const isLargeBoat = (boat: LargeBoat | Boat): boat is LargeBoat =>
+  (boat as LargeBoat).boatType.id === LARGE_BOAT_ID;
+
+export const getCertificate = (
+  certificates: BoatCert[],
+  certificateType: BoatCertificateType
+): BoatCert => {
+  return certificates.filter(
+    cert => cert.certificateType === certificateType
+  )[0];
+};
+
+export const getFilename = (url: string | null): string | undefined => {
+  if (!url) {
+    return undefined;
+  }
+
+  return url.split('/').pop();
+};
+
+export const getValidUntilColor = (
+  validUntil: string
+): 'critical' | 'standard' => {
+  const isBeforeToday =
+    new Date(validUntil) < new Date(new Date().toDateString());
```

Comment:
Isn't this enough?
```suggestion
    new Date(validUntil) < new Date();
```

Reply:
This is for omitting the time in the "before check". With the code you suggested the valid until with current date would resolve as "not valid anymore" and I assume that the date is "inclusive" (it's still valid at the last day). But agree that might raise eyebrows and if we will start doing more date operations we should start using a a proper date library for that.